src/services/i18n.ts

```diff
@@ -0,0 +1,25 @@
+import i18n from 'i18next'
+import i18nLanguageDetector from 'i18next-browser-languagedetector'
+
+import packageJson from '../../package.json'
+import isDev from 'services/is-dev'
+
+const resources = {
+  en: { translation: require('i18n/en.json') },
```

Comment:
can we use translation namespaces?
`common` for standard terms (e.g. `save`)
`glossary` for branding (e.g. `Care Team Member`)
`translation` for the rest

and possibly a namespace specific to `steps` ?

Reply:
I'm down, but the docs aren't super clear on how to set it up. Do you have an example handy?